hicommonwealth/commonwealth
# #5477 - Create Vote Weight Module

This PR completes ticket #5477, which asks for the creation of a vote weight module to be placed in the side bar navigation. The relevant designs for both desktop and mobile can be found [here](https://www.figma.com/file/KOEVNGXStBhmX3NnYq7MxP/Community-Staking?type=design&node-id=1153-224753&mode=design&t=BVH8vX8wmVfLSNNr-4).

## Link to Issue
Closes: #5477 

## Description of Changes
- creates a module listing the stake a user has in a community and the value of that stake in the relevant denomination
- the module includes an informative popover that appears upon hovering over the 'info' icon on desktop, and upon tap on mobile devices
- the aforementioned popover closes when upon clicking the 'x' in the upper right hand corner on mobile

## Test Plan
- navigate to the `/components` page and scroll to the Vote Weight Module section
- check that expected information is within module
- hover over info button to see info popover
- use mobile view on your browser to tap on info button. Also check that info popover has close button and that it closes the popover
